inputs/simple-list.vue

```diff
@@ -233,14 +234,47 @@
         }
       },
       removeItem(index) {
-        this.items.splice(index, 1);
+        const [removedItem] = this.items.splice(index, 1);
+
+        this.removedItem = removedItem;
+
         this.update(this.items);
 
         if (this.items.length) { // you always select before deleting
           this.selectItem(this.currentItem - 1);
         } else {
           this.currentitem = null;
         }
+
+        if (this.args.autocomplete && this.args.autocomplete.allowRemove) {
+          const listName = this.args.autocomplete.list;
+
+          return this.$store.dispatch('updateList', {
+            listName: listName,
+            fn: this.handleRemoveItem
+          });
+        }
+      },
+      /**
+       * Handles remove item from list
+       * @param {Array.<Object>} items
+       */
+      handleRemoveItem(items = []) {
+        // validate that the list has items with these properties
+        const stringProperty = getProp(items, 'text'),
+          countProperty = getProp(items, 'count');
+
+        if (stringProperty && countProperty) {
+          const itemIndex = getItemIndex(items, (this.removedItem || {}).text, 'text');
```

Comment:
If you set `this.removedItem` to an empty object when you delete it and by default, you will never have to do this validation

Reply:
I'm taking this suggestion, however, since we don't have control over the lists, I understand this is something worth checking.